x-pack/filebeat/input/cel/input.go

```diff
@@ -483,6 +513,12 @@ func (i input) run(env v2.Context, src *source, cursor map[string]interface{}, p
 	return err
 }
 
+func updateStatus(reporter status.StatusReporter, state status.Status, msg string) {
+	if reporter != nil {
+		reporter.UpdateStatus(state, msg)
```

Comment:
💡 idea (not for this PR): Most of the Filebeat inputs expose stats on `/inputs/`. We could simultaneously push a `string` metric containing the status and message. This would help users of standalone Filebeat.

Reply:
sounds like a good idea, where do we write down these ideas?! because after this PR we need to do the same status reporting for all remaining v2.Plugin-based Filebeat inputs 😄

Reply:
This was originally being tracked outside of this repo. I think it makes a lot sense for us to have a public issue in elastic/beats to track the other work. That idea (if it makes sense) could go into that "epic".